### PR TITLE
[Reviews-40] Pagination

### PR DESCRIPTION
**Pull request by aurorahey**
#### What problem is this solving?

admin panel pagination bugfix

#### How to test it?

Please test this in the workspace, and contact me if it is unlinked
[https://aurora--sandboxusdev.myvtex.com/admin/reviews-ratings/pending/](url)

### CODE REVIEW

File: dotnet/Services/ProductReviewService.cs [hunk 1]
```diff
@@ -301,7 +301,7 @@ public async Task<ReviewsResponseWrapper> GetReviews(string searchTerm, int from
 
             string sortQuery = await this.GetSortQuery(orderBy);
 
-            ReviewsResponseWrapper wrapper = await _productReviewRepository.GetProductReviewsMD($"{searchQuery}{sortQuery}{statusQuery}");
+            ReviewsResponseWrapper wrapper = await _productReviewRepository.GetProductReviewsMD($"{searchQuery}{sortQuery}{statusQuery}", Convert.ToString(from), Convert.ToString(to));
```
Comment (btalma) [inline dotnet/Services/ProductReviewService.cs:304]: ReviewsResponseWrapper wrapper = await _productReviewRepository.GetProductReviewsMD($"{searchQuery}{sortQuery}{statusQuery}", from.ToString(), to.ToString());
is also valid.

File: dotnet/DataSources/IProductReviewRepository.cs [hunk 2]
```diff
@@ -25,7 +25,7 @@ public interface IProductReviewRepository
         Task<string> VerifySchema();
         Task<string> VerifyMigration();
         Task<string> SuccessfulMigration();
-        Task<ReviewsResponseWrapper> GetProductReviewsMD(string searchQuery, string from = null, string to = null);
+        Task<ReviewsResponseWrapper> GetProductReviewsMD(string searchQuery, string from, string to);
```
Comment (btalma) [inline dotnet/DataSources/IProductReviewRepository.cs:28]: Setting these parameters to null makes them optional.  That way we can avoid having to set them to null in the cases where the pagination is not specified.

**Review by btalma — APPROVED** [review]
Comments are educational, not required changes.

File: dotnet/DataSources/IProductReviewRepository.cs (hunk 2 above)
Comment (aurorahey) [reply dotnet/DataSources/IProductReviewRepository.cs:28]: Thank you for the information

**Pull request merged**